docs/api/buckets.rst

```diff
@@ -74,14 +75,17 @@ Creating a bucket
         request header can be provided. A ``412 Precondition Failed`` error response
         will be returned if the record already exists.
 
-Retrieve an already existing bucket
-===================================
+
+.. _bucket-get:
+
+Retrieve an existing bucket
+===========================
 
 .. http:get:: /buckets/(bucket_id)
 
-    **Requires authentication**
+    :synopsis: Returns a specific bucket by its id.
```

Comment:
In English, "id" is a psychoanalysis term (see: Freud). You want ID, which means "identification" or "identifier". :smile: